otelcollector/otel-config.yaml

```diff
@@ -58,3 +58,7 @@ service:
       # Order is important here. Otherwise, the attributes processor will not be able to read the attributes from the auth context.
       processors: [attributes/from_auth_context, batch]
       exporters: [clickhouse]
+  telemetry:
```

Comment:
When the loglevel is `none` the meterProvider will not be initialized and the the prometheus listener will not come up. I tested this on my local setup.
 
In general this has to be set. Leaving it blank results in an error (sthg like "" is not a proper value). 

To handle this gracefully, I added a check in the helm chart and override this value accordingly, the listen address is then as well not set and the prometheus server will simply not start and nothing in kubernetes will point to the endpoint. 
It is "gracefully" handled by setting sane defaults here otherwise we might need multiple configs or some preprocessing of the config file which is less predictive.

Reply:
ok, thank you